tap_zendesk/discover.py

```diff
@@ -20,12 +24,41 @@ def load_shared_schema_refs():
 
     return shared_schema_refs
 
-def discover_streams(client):
+def discover_streams(client, config):
     streams = []
+    error_list = []
     refs = load_shared_schema_refs()
 
+
     for s in STREAMS.values():
-        s = s(client)
+        s = s(client, config)
         schema = singer.resolve_schema_references(s.load_schema(), refs)
+        try:
+            # Here it call the check_access method to check whether stream have read permission or not.
+            # If stream does not have read permission then append that stream name to list and at the end of all streams
+            # raise forbidden error with proper message containing stream names.
+            s.check_access()
+        except ZendeskForbiddenError as e:
+            error_list.append(s.name) # Append stream name to the error_list
+        except zenpy.lib.exception.APIException as e:
+            args0 = json.loads(e.args[0])
+            err = args0.get('error')
+
+            if isinstance(err, dict):
```

Comment:
@prijendev Can you add same comment here as it is added in this PR - https://github.com/singer-io/tap-zendesk/pull/69/files#diff-89887ca52395ce152e9723c5595655edcd41793204060336e88ea9805c274433R142